enhancements/etcd/protecting-etcd-quorum-during-control-plane-scaling.md

```diff
@@ -105,7 +105,7 @@ This enhancement does not introduce any new API extensions.
 #### Protecting etcd during scaling operations
 
 To ensure that the safety requirements described above are maintained during scaling operations,
-the etcd Operator will manage the etcd cluster in clusters with a
+the etcd operator will manage the etcd cluster in clusters with a
```

Comment:
nit: Operator is written with capital O throughout the doc. It just hurts my eyes a little :-)

Reply:
Will fixup :)

Reply:
I remember now, this was an artifact of me not know how etcd should be written, I had variation on Etcd Operator which looked ok to me but then i was told etcd had to be all lower, which I fixed, but forgot the O 😅